results/intfloat__multilingual-e5-small/IndicNLPNewsClassification.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
I wasn't expecting results to be this high for low resource languages. Maybeit's due to the small number of samples selected in the subsampling function. Can you please run this again with 2048 samples instead?